src/crab/splitdbm/graph_views.hpp

```
@@ -0,0 +1,438 @@
+        AdjList(const R& _rG, const VertId _v_ex) : rG(_rG), v_ex(_v_ex) {}
+        iterator begin() const { return iterator(rG.begin(), v_ex); }
+        iterator end() const { return iterator(rG.end(), v_ex); }
+
+      private:
+        R rG;
+        VertId v_ex;
+    };
+    using NeighbourConstRange = AdjList<GNeighbourConstRange, AdjIterator<typename GNeighbourConstRange::iterator>>;
+    using ENeighbourConstRange = AdjList<GENeighbourConstRange, EAdjIterator<typename GENeighbourConstRange::iterator>>;
+
+    NeighbourConstRange succs(VertId v) const {
```

Clang (14.0.6)

[warning] use a trailing return type for this function

(modernize-use-trailing-return-type)